README.md

```diff
@@ -102,10 +102,11 @@ If the `CONFIG_FILE` environment variable is not set, the program will fall back
    - `CONFIG_NETWORK_NAME` - Set this to your network name.
 
 2. **Whitelist Configuration**:
+   - `GOOGLE_APPLICATION_CREDENTIALS` - set path to `minasheets.json` file including credentials to connect to Google Sheets.
```

Comment:
[nit] this is a gcp service account json used to access google spreadsheets.  The name `minasheets.json` does not reflect that. gcp-sa.json or gcp-sa-minasheets.json I believe would make more sense. But that is not a blocker on my side.